### PR TITLE
chore(traces): use debug/display shortcuts on trace events

### PR DESCRIPTION
**Pull request by fanatid**
When worked on ~#3032~ #4243  noticed that [tracing::field::debug](https://tracing.rs/tracing/field/fn.debug.html) / [tracing::field::display](https://tracing.rs/tracing/field/fn.display.html) used instead shortcuts `?`/`%` on creating trace events. This pull request replace methods by shortcuts.

### CODE REVIEW

File: src/sinks/gcp/cloud_storage.rs [hunk 1]
```diff
@@ -318,8 +317,8 @@ impl RequestWrapper {
 
         debug!(
             message = "sending events.",
-            bytes = &field::debug(body.len()),
-            key = &field::debug(&key)
+            bytes = ?body.len(),
+            key = ?key
```
Comment (bruceg) [inline src/sinks/gcp/cloud_storage.rs:321]: Could this be the even shorter `?key`?

File: src/sources/docker.rs [hunk 2]
```diff
@@ -349,8 +348,8 @@ impl DockerSource {
 
                 trace!(
                     message = "found already running container.",
-                    id = field::display(&id),
-                    names = field::debug(&names)
+                    id = %id,
```
Comment (bruceg) [inline src/sources/docker.rs:351]: Ditto short form

File: src/sources/docker.rs [hunk 3]
```diff
@@ -369,7 +368,7 @@ impl DockerSource {
                         }
                     }),
                 ) {
-                    trace!(message = "container excluded.", id = field::display(&id));
+                    trace!(message = "container excluded.", id = %id);
```
Comment (bruceg) [inline src/sources/docker.rs:371]: Ditto

**Review by bruceg — APPROVED** [review]
Other than not using the even shorter form of `name = %name` => `%name%`, this looks fine.

**Comment by fanatid** [comment]
Thanks @bruceg! Fixed.

**Review by Hoverbear — APPROVED** [review]

**Pull request merged**